docs/helpers/users.json

```diff
@@ -13,5 +13,9 @@
     "eng": "People",
     "cop": "Pilaoc",
     "ara": "الشعب"
-  }
-}
\ No newline at end of file
+  },
+  "reader": {
```

Comment:
fyi you can use these helpers instead of typing it out in full in your documents, e.g.

"user" : "reader"

instead of

+      "user": {
+        "eng": "Reader",
+        "cop": "Pi`anagnwctyc",
+        "ara": "القارئ"
+      }